content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -92,15 +92,16 @@ admins constructing their own audit profiles.
 ## Audit backends
 
 Audit backends persist audit events to an external storage.
-[Kube-apiserver][kube-apiserver] out of the box provides two backends:
+[Kube-apiserver][kube-apiserver] out of the box provides three backends:
 
 - Log backend, which writes events to a disk
 - Webhook backend, which sends events to an external API
 - Dynamic backend, which configures webhook backends through an AuditSink API object.
 
-In both cases, audit events structure is defined by the API in the
+In top two cases, audit events structure is defined by the API in the
 `audit.k8s.io` API group. The current version of the API is
-[`v1`][auditing-api].
+[`v1`][auditing-api]. But in dynamic backend, AuditSink objects are defined in
+`auditregistration.k8s.io` API group. The current version of the API is v1alpha1.
```

Comment:
the auditsink registration is separate from the format of the audit events sent to the sink. in all cases, the events sent are defined by the API in the `audit.k8s.io` API group.

Reply:
Thanks @liggitt , modified.